ui/app/src/views/explore/ExploreView.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The Perses Authors
```

Comment:
UI file archi is a little bit messy, but I think this folder should be moved to `ui/app/src/views/projects`, next to `dashboards` folder :)